src/dune_engine/build_system.ml

```diff
@@ -2389,6 +2389,39 @@ let is_target file =
   let+ targets = targets_of ~dir:(Path.parent_exn file) in
   Path.Set.mem targets file
 
+(* Evaluate a rule without building the action dependencies *)
```

Comment:
This comment seems misleading, since we do build dependencies in `execute_action_stdout`.

Reply:
Indeed, I removed it and updated the comment in the mli

Reply:
same here